src/test/java/org/simulator/sbml/SBMLTestSuiteRunner.java

```diff
@@ -270,7 +270,7 @@ public static void statisticForSolvers(String file, int from, int to)
 
           File f = new File(csvfile);
           if (f.exists()) {
-            inputData = csvimporter.convert(model, csvfile);
+            inputData = csvimporter.readMultiTableFromCSV(model, csvfile);
```

Comment:
Camel case.

Reply:
Here, do you suggest to camel case the `csvimporter` to `csvImporter`?